src/ensembl/src/content/app/browser/browserHelper.ts

```diff
@@ -181,7 +181,7 @@ export const validateRegion = async (params: {
       const regionId = buildEnsObjectId({
         genomeId,
         type: 'region',
-        objectId: regionInput
+        objectId: response.region_id || ''
```

Comment:
the `''` would surely create an invalid ens object id.

I understand that this is because of the way we defined the response type. We should either convince typescript that `response.region_id` at this point will always be `string` (`objectId: response.region_id as string`), or clarify what the possible response shapes are and update our types.

Reply:
I've updated it to `objectId: response.region_id as string` for now.

Reply:
👌